### PR TITLE
[MIG] [18.0] partner_credit_limit: migrated module

### PR DESCRIPTION
**Pull request by ShwetaSerpentCS**
Responsible : Jayram

### CODE REVIEW

File: partner_credit_limit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Partner Credit Limit',
+    'version': '18.0.1.0.0',
+    'category': 'Partner',
+    'license': 'LGPL-3',
+    'author': 'Tiny, Serpent Consulting Services Pvt. Ltd.',
+    'website': 'http://www.serpentcs.com',
```
Comment (Trivedi-Vacha-SerpentCS) [inline partner_credit_limit/__manifest__.py]:
```suggestion
    'website': 'https://www.serpentcs.com',
```

File: partner_credit_limit/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    'maintainer': 'Serpent Consulting Services Pvt. Ltd.',
+    'summary': 'Set credit limit warning',
+    'depends': [
+        'sale_management',
+    ],
+    'data': [
+        'views/partner_view.xml',
+    ],
+    'installable': True,
+    'auto_install': False,
```
Comment (Trivedi-Vacha-SerpentCS) [inline partner_credit_limit/__manifest__.py]: default value is False, no need to define

File: partner_credit_limit/models/sale.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            order.check_limit()
+        return res
+
+    @api.constrains('amount_total')
+    def check_amount(self):
+        for order in self:
+            order.check_limit()
+
+    @api.model
+    def create(self, vals):
```
Comment (Trivedi-Vacha-SerpentCS) [inline partner_credit_limit/models/sale.py]:
```
@api.model_create_multi
    def create(self, vals_list):
```

**Review by Trivedi-Vacha-SerpentCS — CHANGES_REQUESTED** [review]

**Comment by ketanserpentcs** [comment]
Testing is assigned to Devraj

**Comment by ketanserpentcs** [comment]
QA done with index by Devraj

**Comment by Trivedi-Vacha-SerpentCS** [comment]
@ShwetaSerpentCS kindly verify index and banner

**Comment by ShwetaSerpentCS** [comment]
> @ShwetaSerpentCS kindly verify index and banner

Done. Ready to merge. @Trivedi-Vacha-SerpentCS

**Comment by JayVora-SerpentCS** [comment]
Do we need this module?  
ping me on skype

**Pull request merged**